nipoppy/data/layouts/layout-0.2.x.json

```diff
@@ -0,0 +1,98 @@
+        "description": "This directory is for downloaded data archives (e.g., tarballs or zipped files)."
+    },
+    "dpath_code": {
+        "path": "code",
+        "description": "This directory is for custom code."
+    },
+    "dpath_proc": {
+        "path": "proc",
+        "description": "This directory is for storing the Nipoppy configuration information and custom code."
+    },
+    "dpath_containers": {
+        "path": "proc/containers",
+        "description": "This directory is for storing the container image files for the BIDS conversion/processing pipelines."
+    },
+    "dpath_descriptors": {
+        "path": "proc/descriptors",
+        "description": "This directory is for storing Boutiques descriptors for user-added pipelines"
+    },
+    "dpath_invocations": {
+        "path": "proc/invocations",
+        "description": "This directory is for storing Boutiques invocations for processing pipelines."
+    },
+    "dpath_tracker_configs": {
+        "path": "proc/tracker_configs",
+        "description": "This directory is for storing tracker configurations for processing pipelines."
+    },
+    "dpath_pybids_db": {
+        "path": "proc/pybids/bids_db",
+        "description": "This directory is for PyBIDS databases generated/used during processing."
+    },
+    "dpath_pybids_ignore_patterns": {
+        "path": "proc/pybids/ignore_patterns",
+        "description": "This directory is for storing ignore patterns to be used by PyBIDS during dataset indexing."
+    },
+    "dpath_scratch": {
+        "path": "scratch",
+        "description": "This directory is for scratch data."
+    },
+    "dpath_to_reorg": {
+        "path": "scratch/raw_imaging",
+        "description": "This directory is for raw, unorganized imaging file dumps."
+    },
+    "dpath_logs": {
+        "path": "scratch/logs",
+        "description": "This directory contains log files."
+    },
+    "dpath_tabular": {
+        "path": "tabular",
+        "description": "This directory is for tabular/phenotypic data."
+    },
+    "dpath_assessments": {
+        "path": "tabular/assessments",
+        "description": "This directory is for assessments data."
+    },
+    "fpath_config": {
+        "path": "global_config.json",
+        "description": "The configuration file for a dataset is where all data processing is configured."
+    },
+    "fpath_doughnut": {
+        "path": "scratch/raw_imaging/doughnut.csv",
+        "description": "The doughnut file is generated/updated internally and is used to keep track of the raw data organization and BIDS conversion process."
+    },
+    "fpath_manifest": {
+        "path": "manifest.csv",
+        "description": "The manifest file lists all expected study participants and visits."
+    },
+    "fpath_imaging_bagel": {
+        "path": "derivatives/bagel.csv",
+        "description": "The imaging bagel file lists availability of derived imaging data and pipeline processing statuses."
+    }
```

Comment:
These should be changed to TSV in this PR or #381 (depending on which PR is merged first)